frontend/pages/home.tsx

```diff
@@ -4,39 +4,58 @@ import styled from 'styled-components';
 import Image from 'next/image';
 import homepagePic from '../public/home-image.png';
 import Layout from '../components/Layout';
+import { colors, size, maxDevice } from '../templates/mediaSizes';
 
 const HomeContainer = styled.div`
-  background-color: #ef8b6f;
   display: flex;
   flex-direction: column;
-  min-height: 100vh;
   padding: 2rem;
+  background-color: ${colors.homepageBackground};
+  .title-below {
+    #mi
+    @media min-width: ${size.laptop + 1}px {
+      display: block;
+    }
+  }
+  .title-above {
+    @media ${maxDevice.laptop} {
+      display: block;
+    }
+  }
 `;
 
 const HomeTitle = styled.h1`
   font-family: din-2014;
+  display: none;
 `;
 
 const Row = styled.div`
   display: flex;
   flex-direction: row;
-  justify-content: space-between;
+  justify-content: flex-start;
+  flex-wrap: wrap;
   padding-top: 1rem;
   width: 100%;
 `;
 
 const ImageContainer = styled(Row)`
   margin: 1rem 0rem 1rem 0rem;
+  @media ${maxDevice.laptop} {
+    height: auto;
+    max-width: 600px;
+  }
 `;
+const ImageContent = styled(Image)``;
```

Comment:
kinda bothers me that we have to scroll vertically when we enter "tablet" size territory
could we set a max-height, probably based on `vh` or `vw`

Reply:
here's what I'm seeing:

when I'm in mobile view, its fine, no need to scroll vertically:
<img width="508" alt="Screen Shot 2022-02-22 at 1 42 58 PM" src="https://user-images.githubusercontent.com/17476012/155198070-362d991a-9f0d-4aa3-aed4-f6c5bdb9c780.png">

when I extend the window width, the image gets larger, so I have to scroll vertically:
<img width="769" alt="Screen Shot 2022-02-22 at 1 43 11 PM" src="https://user-images.githubusercontent.com/17476012/155198239-e49dc0ab-454d-405a-a1ab-04f6d9a1607a.png">

ideally, there's some solution that allows me to extend the width of my screen, without having to scroll down to see the bottom of the page

Reply:
fixed!